birdhouse/read-configs.include.sh

```diff
@@ -470,8 +497,9 @@ _read_basic_configs_pre() {
 
 
 _read_basic_configs_post() {
-    read_env_local  # again to override components default.env, need discover_env_local
-    set_old_backwards_compatible_variables  # after read_env_local to use updated value and not default value
+    read_env_local  # override default env, needs discover_env_local to run first
```

Comment:
env local always override any default env, that's a given.  The most important info in the comment was that it is read **again** (so any append operation will be repeated again) and this time it is meant to override the **components** default env (while the first time it only override the **default** default.env).

Can you restore the original comment to preserve the clarification?